src/components/SamplePanel/index.js

```diff
@@ -140,15 +148,29 @@ const SamplePanel = ({sampleName, sampleData, config, reference, socket, panelEx
         }
         />
     )*/
+    mutationsTree: (
+      <MutationsTree
+       data={sampleVariant}
+       renderProp={ showSinglePanel === "mutationsTree" ?
+            (<ContractChart handleClick={() => goToChart(false)}/>) :
```

Comment:
I am not sure whether this is formatted in line with the rest of the project but usually it is easier to read
```
condition
? (
  A
) : (
  B
)
```